tests/pl/light_HassLightSet.yaml

```diff
@@ -35,3 +60,4 @@ tests:
       slots:
         color: "white"
         area: "bathroom"
+    response: Ustawiono kolor w łazience na białe
```

Comment:
Może "Ustawiono kolor w łazience na biały"?

Reply:
The sam problem which I describe above.. we ask `Zmień kolory w łazience na białe` or `Zmień wszystkie kolory w łazience na białe` but I can't on the fly replace color to different naming `biały `. We can replace color name on all sentences to eg. `Zmień wszystkie kolory w łazience na biały` then the response will look differently.